<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplexTextDialog</class>
 <widget class="QDialog" name="ComplexTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор текста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="HelperLibraryNS::ComplexTextEditor" name="complexTextEditor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="toolTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="statusTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Применить и закрыть</string>
       </property>
       <property name="text">
        <string>Применить и закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="toolTip">
        <string>Закрыть без сохранения изменений</string>
       </property>
       <property name="statusTip">
        <string>Закрыть без сохранения изменений</string>
       </property>
       <property name="whatsThis">
        <string>Закрыть без сохранения изменений</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelperLibraryNS::ComplexTextEditor</class>
   <extends>QWidget</extends>
   <header>HelperLibrary/Widgets/Text/complextexteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
